docpages/example_programs/interactions_and_components/commandhandler.md

```diff
@@ -1,13 +1,13 @@
 \page commandhandler Using a command handler object
 
 If you have many commands in your bot, and want to handle commands from multiple sources (for example modern slash commands, and more regular
-prefixed channel messages) you should consider instantiating a dpp::commandhandler object. This object can be used to automatically route
+prefixed channel messages) you should consider instantiating a `dpp::commandhandler` object. This object can be used to automatically route
 commands and their parameters to functions in your program. A simple example of using this object to route commands is shown below, and will
 route both the /ping (global slash command) and .ping (prefixed channel message command) to a lambda where a reply can be generated.
 
-\note	This example automatically hooks the dpp::cluster::on_message_create and dpp::cluster::on_slashcommand events. This can be overridden if needed to allow you to still make use of these functions for your own code, if you need to do this please see the constructor documentation for dpp::commandhandler.
+\note	This example automatically hooks the `dpp::cluster::on_message_create` and `dpp::cluster::on_slashcommand events`. This can be overridden if needed to allow you to still make use of these functions for your own code, if you need to do this please see the constructor documentation for `dpp::commandhandler`.
```

Comment:
`dpp::cluster::on_slashcommand events` -> `dpp::cluster::on_slashcommand` events

Note that generally doxygen will auto create links to the docs for recognised symbols like dpp::cluster::on_slashcommand, check that by putting the backticks around them you arent breaking this or making it look weird